lib/cppcheck.cpp

```diff
@@ -520,6 +520,7 @@ unsigned int CppCheck::checkClang(const FileWithDetails &file)
             fdump << "    <c version=\"" << mSettings.standards.getC() << "\"/>\n";
             fdump << "    <cpp version=\"" << mSettings.standards.getCPP() << "\"/>\n";
             fdump << "  </standards>\n";
+            fdump << "  <library lib="<< mSettings.getLib() << "/>\n";
```

Comment:
I suggest that we don't try to encode the libraries into a string. I would prefer to have a `<library lib="..."/>` for each library.

The std.cfg is not necessary to mention in the dumpfile though.